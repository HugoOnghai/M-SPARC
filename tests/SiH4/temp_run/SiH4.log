
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat SiH4.inpt
# nprocs: 1
# Test: SiH4 #
CELL: 23 23 23
MESH_SPACING: 0.25
BOUNDARY_CONDITION: 1
RHO_TRIGGER: 4
EXCHANGE_CORRELATION: LDA_PW
MAXIT_SCF: 30
TOL_SCF: 1e-6
PRINT_FORCES: 1
PRINT_ATOMS: 1
<\INPT>

Warning: "BOUNDARY_CONDITION" is obsolete, use "BC" instead!
 Reading .ion file ...
 Number of atom types : 2
 Total number of atoms: 5
 Default atomic mass for Si is 28.085000
 Default atomic mass for H is 1.007975
 pseudo_dir # 1 = Si.psp8
 pseudo_dir # 2 = H.psp8

 COORD:
  11.500000000000000  11.500000000000000  11.500000000000000
  13.127432021000001  13.127432021000001  13.127432021000001
   9.872567978999999   9.872567978999999  13.127432021000001
   9.872567978999999  13.127432021000001   9.872567978999999
  13.127432021000001   9.872567978999999   9.872567978999999

   0.030168927102176

   2.303934532781000

 kpoint grid before symmetry:
     0     0     0

 kpoint grid after symmetry:
     0     0     0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 30
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 10
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (8.740 sec)

 Estimated memory usage:
 Total:    3.12 GB
 orbitals            :  245.47 MB
 sparse matrices     :    1.97 GB
 global-size vectors :  112.41 MB
 spherical harmonics :  733.47 MB
 mixing histories    :   85.91 MB

 Finding rb for Si ...
 rb = {4.000 4.000 4.000}, int_b = -4.000017124039084, err_rb = 1.712e-05
 rb = {4.250 4.250 4.250}, int_b = -4.000015193321995, err_rb = 1.519e-05
 rb = {4.500 4.500 4.500}, int_b = -4.000015343112957, err_rb = 1.534e-05
 rb = {4.750 4.750 4.750}, int_b = -4.000012933723863, err_rb = 1.293e-05
 rb = {5.000 5.000 5.000}, int_b = -4.000012318580879, err_rb = 1.232e-05
 rb = {5.250 5.250 5.250}, int_b = -4.000010249749264, err_rb = 1.025e-05
 rb = {5.500 5.500 5.500}, int_b = -4.000010257937611, err_rb = 1.026e-05
 rb = {5.750 5.750 5.750}, int_b = -4.000010112668022, err_rb = 1.011e-05
 rb = {6.000 6.000 6.000}, int_b = -3.999998671734363, err_rb = 1.328e-06
 rb = {6.250 6.250 6.250}, int_b = -4.000000276001368, err_rb = 2.760e-07
 rb = {6.500 6.500 6.500}, int_b = -3.999999945892093, err_rb = 5.411e-08
 rb = {6.750 6.750 6.750}, int_b = -4.000000007608960, err_rb = 7.609e-09
 rb = {6.750 6.750 6.750}
 Finding rb for H ...
 rb = {1.500 1.500 1.500}, int_b = -1.000014336781230, err_rb = 1.434e-05
 rb = {1.750 1.750 1.750}, int_b = -1.000007286119245, err_rb = 7.286e-06
 rb = {2.000 2.000 2.000}, int_b = -1.000006366997262, err_rb = 6.367e-06
 rb = {2.250 2.250 2.250}, int_b = -1.000005476255216, err_rb = 5.476e-06
 rb = {2.500 2.500 2.500}, int_b = -1.000004874337470, err_rb = 4.874e-06
 rb = {2.750 2.750 2.750}, int_b = -1.000004484923488, err_rb = 4.485e-06
 rb = {3.000 3.000 3.000}, int_b = -1.000004263660449, err_rb = 4.264e-06
 rb = {3.250 3.250 3.250}, int_b = -1.000004170905200, err_rb = 4.171e-06
 rb = {3.500 3.500 3.500}, int_b = -1.000004606980896, err_rb = 4.607e-06
 rb = {3.750 3.750 3.750}, int_b = -1.000005438528102, err_rb = 5.439e-06
 rb = {4.000 4.000 4.000}, int_b = -1.000007542262380, err_rb = 7.542e-06
 rb = {4.250 4.250 4.250}, int_b = -1.000007872808125, err_rb = 7.873e-06
 rb = {4.500 4.500 4.500}, int_b = -1.000009594584392, err_rb = 9.595e-06
 rb = {4.750 4.750 4.750}, int_b = -1.000008644655050, err_rb = 8.645e-06
 rb = {5.000 5.000 5.000}, int_b = -1.000009066855449, err_rb = 9.067e-06
 rb = {5.250 5.250 5.250}, int_b = -1.000007972313449, err_rb = 7.972e-06
 rb = {5.500 5.500 5.500}, int_b = -1.000008589128159, err_rb = 8.589e-06
 rb = {5.750 5.750 5.750}, int_b = -1.000008963291194, err_rb = 8.963e-06
 rb = {6.000 6.000 6.000}, int_b = -0.999998818567388, err_rb = 1.181e-06
 rb = {6.250 6.250 6.250}, int_b = -1.000000245479738, err_rb = 2.455e-07
 rb = {6.500 6.500 6.500}, int_b = -0.999999951975485, err_rb = 4.802e-08
 rb = {6.750 6.750 6.750}, int_b = -1.000000006728957, err_rb = 6.729e-09
 rb = {6.750 6.750 6.750}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 7.999999932466

 Integration b_ref = 8.000000000006

 ****************************************
 *          Eself_ref = 35.612075       *
 ****************************************
 Done. (0.396156 s)
 Time for b calculation: 0.410 seconds.

 Starting calculating nonlocal projectors ...
 Done. (0.155656 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 7.21642e-09 in 187 iterations.
 Poisson problem took 29.286634s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 2.951663
 This SCF iteration took 22.949 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.375530
 This SCF iteration took 12.971 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = -0.195933
 This SCF iteration took 12.970 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -0.370383
 AAR converged to a relative residual of 8.23045e-09 in 151 iterations.
 Poisson problem took 23.873332s
 Escc = 1.01089779
 Etot = -6.20285840
 Eatom = -1.24057168
 Error in SCF iteration: 1.9127e-01 
 This SCF iteration took 38.019 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = -0.338860
 AAR converged to a relative residual of 9.93056e-09 in 137 iterations.
 Poisson problem took 21.529408s
 Escc = 0.63456995
 Etot = -6.23176491
 Eatom = -1.24635298
 Error in SCF iteration: 9.4985e-02 
 This SCF iteration took 35.856 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = -0.307673
 AAR converged to a relative residual of 9.22158e-09 in 133 iterations.
 Poisson problem took 21.023649s
 Escc = 0.27616839
 Etot = -6.23796556
 Eatom = -1.24759311
 Error in SCF iteration: 4.5614e-02 
 This SCF iteration took 35.271 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = -0.286710
 AAR converged to a relative residual of 6.68694e-09 in 127 iterations.
 Poisson problem took 20.134466s
 Escc = 0.03681919
 Etot = -6.23948180
 Eatom = -1.24789636
 Error in SCF iteration: 1.4362e-02 
 This SCF iteration took 34.464 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = -0.283261
 AAR converged to a relative residual of 8.4534e-09 in 103 iterations.
 Poisson problem took 16.588498s
 Escc = -0.00229003
 Etot = -6.23955656
 Eatom = -1.24791131
 Error in SCF iteration: 7.4831e-03 
 This SCF iteration took 30.940 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = -0.283686
 AAR converged to a relative residual of 8.52783e-09 in 97 iterations.
 Poisson problem took 15.720004s
 Escc = 0.00381359
 Etot = -6.23955661
 Eatom = -1.24791132
 Error in SCF iteration: 1.5675e-03 
 This SCF iteration took 30.176 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = -0.283336
 AAR converged to a relative residual of 9.87209e-09 in 93 iterations.
 Poisson problem took 15.384126s
 Escc = -0.00221379
 Etot = -6.23955892
 Eatom = -1.24791178
 Error in SCF iteration: 1.0375e-03 
 This SCF iteration took 29.631 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = -0.283525
 AAR converged to a relative residual of 5.49296e-09 in 85 iterations.
 Poisson problem took 14.047990s
 Escc = -0.00085675
 Etot = -6.23955901
 Eatom = -1.24791180
 Error in SCF iteration: 5.0441e-04 
 This SCF iteration took 28.245 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = -0.283619
 AAR converged to a relative residual of 5.24725e-09 in 79 iterations.
 Poisson problem took 13.053787s
 Escc = 0.00024614
 Etot = -6.23955902
 Eatom = -1.24791180
 Error in SCF iteration: 1.1840e-04 
 This SCF iteration took 27.164 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = -0.283594
 AAR converged to a relative residual of 9.85633e-09 in 50 iterations.
 Poisson problem took 8.793083s
 Escc = 0.00004108
 Etot = -6.23955903
 Eatom = -1.24791181
 Error in SCF iteration: 2.6981e-05 
 This SCF iteration took 23.106 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = -0.283593
 AAR converged to a relative residual of 9.92786e-09 in 32 iterations.
 Poisson problem took 6.168194s
 Escc = 0.00002997
 Etot = -6.23955902
 Eatom = -1.24791180
 Error in SCF iteration: 2.2713e-05 
 This SCF iteration took 20.438 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = -0.283590
 AAR converged to a relative residual of 9.68778e-09 in 31 iterations.
 Poisson problem took 6.044726s
 Escc = -0.00000937
 Etot = -6.23955904
 Eatom = -1.24791181
 Error in SCF iteration: 5.5557e-06 
 This SCF iteration took 20.188 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = -0.283590
 AAR converged to a relative residual of 9.62249e-09 in 14 iterations.
 Poisson problem took 3.534700s
 Escc = -0.00000122
 Etot = -6.23955901
 Eatom = -1.24791180
 Error in SCF iteration: 1.2838e-06 
 This SCF iteration took 17.810 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = -0.283590
 AAR converged to a relative residual of 6.7047e-09 in 13 iterations.
 Poisson problem took 3.396299s
 Escc = -0.00000001
 Etot = -6.23955902
 Eatom = -1.24791180
 Error in SCF iteration: 3.5198e-07 
 This SCF iteration took 17.463 s.


 Finished SCF iteration in 14 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -6.239559015 Ha.       *
 *          Energy per atom = -1.247911803 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 0.765 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
   1.0e-03 *

   0.000023711259322   0.000063462980626   0.000041833461408
  -0.280261102392916  -0.280274013693248  -0.280266066689707
   0.280192216291595   0.280236753856460  -0.280198993336057
   0.280257160697290  -0.280249758222029   0.280230818461875
  -0.280211985855290   0.280223555078191   0.280192408102481

 Max magnitude of forces (Ha/Bohr):     4.854367892979832e-04


 Time for calculating forces: 4.913636 s.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
 11.500000 	 11.500000 	 11.500000 
 13.127432 	 13.127432 	 13.127432 
  9.872568 	  9.872568 	 13.127432 
  9.872568 	 13.127432 	  9.872568 
 13.127432 	  9.872568 	  9.872568 

 Run-time of the program: 498.152089 seconds

ans = 

  struct with fields:

                  cell_typ: 1
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 1
                        L1: 23
                        L2: 23
                        L3: 23
                        Nx: 93
                        Ny: 93
                        Nz: 93
                         N: 804357
                      ecut: -1
              mesh_spacing: 0.250000000000000
                   kptgrid: [0 0 0]
                  kptshift: [0 0 0]
                      nkpt: [1 1 1]
                     tnkpt: 1
                      wkpt: 1
                        BC: 1
                       BCx: 1
                       BCy: 1
                       BCz: 1
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 10
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 1
                      Temp: 2.320903624310025e+03
                       bet: 1.360569312299400e+02
                       npl: 30
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 0
                 Nelectron: 8
                     n_typ: 2
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 30
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 6.250000000000000e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 0
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 1
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'SiH4'
            inputfile_path: '.'
                        XC: 'LDA_PW'
                       Atm: [1x2 struct]
                     n_atm: 5
                     Atoms: [5x3 double]
           mvAtmConstraint: [5x3 double]
                    IsFrac: [2x1 double]
                    IsSpin: [2x1 double]
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [5x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.250000000000000
                        dy: 0.250000000000000
                        dz: 0.250000000000000
                        dV: 0.015625000000000
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [804357x1 double]
                    RR_AUG: [1157625x1 double]
                 RR_AUG_3D: [105x105x105 double]
                      isIn: [105x105x105 logical]
                        RR: [804357x1 double]
                     l_cut: 6
                        SH: [1x7 struct]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                      I_11: [1167x1 double]
                     II_11: [1167x1 double]
                      V_11: [1167x1 double]
                      I_22: [1167x1 double]
                     II_22: [1167x1 double]
                      V_22: [1167x1 double]
                      I_33: [1167x1 double]
                     II_33: [1167x1 double]
                      V_33: [1167x1 double]
                     G_J_1: [9289026x1 double]
                     G_I_1: [9289026x1 double]
                     G_V_1: [9289026x1 double]
                     G_J_2: [9289026x1 double]
                     G_I_2: [9289026x1 double]
                     G_V_2: [9289026x1 double]
                     G_J_3: [9289026x1 double]
                     G_I_3: [9289026x1 double]
                     G_V_3: [9289026x1 double]
                   Lap_std: [804357x804357 double]
                    grad_1: [804357x804357 double]
                    grad_2: [804357x804357 double]
                    grad_3: [804357x804357 double]
                LapPreconL: [804357x804357 double]
                LapPreconU: [804357x804357 double]
              memory_usage: 3.351517784000000e+09
               staticfname: 'SiH4.static'
                 suffixNum: 0
                  outfname: 'SiH4.out'
                  parallel: 0
                         b: [804357x1 double]
                     b_ref: [804357x1 double]
                    rho_at: [804357x1 double]
                     Eself: 19.373491778466668
                 Eself_ref: 35.612074712434790
                       V_c: [804357x1 double]
                 PosCharge: 7.999999932466167
                 NegCharge: -7.999999932466167
                    E_corr: 1.006788423651983e-04
                       rho: [804357x1 double]
                      Atom: [1x5 struct]
                       phi: [804357x1 double]
                       Vxc: [804357x1 double]
                 dvxcdgrho: [804357x1 double]
                      Veff: [804357x1 double]
                         X: [804357x7 double]
                         F: [804357x7 double]
          mixing_hist_fkm1: [804357x1 double]
          mixing_hist_xkm1: [804357x1 double]
                       psi: [804357x10 double]
    upper_bound_guess_vecs: [804357x1 double]
                    EigVal: [10x1 double]
                  lambda_f: -0.283590441783408
                       occ: [10x1 double]
                    Etotal: -6.239559015482858
                     Eband: -2.871130097419907
                       Exc: -2.473591693920420
                    Exc_dc: -3.228787529228508
                  Eelec_dc: 15.249766354519526
                      Eent: -2.021800542846414e-09
                      Escc: -6.244461669102813e-09
                   S_Debug: [1x1 struct]
                     force: [5x3 double]
                 abs_force: [5x1 double]

[Warning: Variable 'ans' was not saved. For variables larger than 2GB use
MAT-file version 7.3 or later.] 
[> In launch_SiH4 (line 5)] 
