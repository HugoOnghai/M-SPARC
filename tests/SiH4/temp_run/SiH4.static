***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
     11.5000000000      11.5000000000      11.5000000000
     13.1274320210      13.1274320210      13.1274320210
      9.8725679790       9.8725679790      13.1274320210
      9.8725679790      13.1274320210       9.8725679790
     13.1274320210       9.8725679790       9.8725679790
Atomic forces (Ha/Bohr):
      0.0000000237       0.0000000635       0.0000000418
     -0.0002802611      -0.0002802740      -0.0002802661
      0.0002801922       0.0002802368      -0.0002801990
      0.0002802572      -0.0002802498       0.0002802308
     -0.0002802120       0.0002802236       0.0002801924
