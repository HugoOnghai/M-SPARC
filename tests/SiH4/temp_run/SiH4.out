***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:09:36                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 23.000000 23.000000 23.000000 
FD_GRID: 92 92 92
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: gaussian
SMEARING: 0.007349864
CHEB_DEGREE: 30
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 30
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: SiH4.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.250000 (Bohr)
Output printed to                  :  SiH4.out
Total number of atom types         :  2
Total number of atoms              :  5
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.75 6.75 6.75
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  H 1
Pseudopotential                    :  H.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.75 6.75 6.75
Number of atoms of type 2          :  4
Estimated total memory usage       :  3.12 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.2405716798E+00        1.913E-01        86.907
2            -1.2463529814E+00        9.498E-02        35.854
3            -1.2475931111E+00        4.561E-02        35.269
4            -1.2478963601E+00        1.436E-02        34.462
5            -1.2479113112E+00        7.483E-03        30.937
6            -1.2479113212E+00        1.568E-03        30.174
7            -1.2479117839E+00        1.038E-03        29.629
8            -1.2479118017E+00        5.044E-04        28.243
9            -1.2479118031E+00        1.184E-04        27.162
10           -1.2479118063E+00        2.698E-05        23.103
11           -1.2479118047E+00        2.271E-05        20.436
12           -1.2479118087E+00        5.556E-06        20.187
13           -1.2479118018E+00        1.284E-06        17.808
14           -1.2479118031E+00        3.520E-07        17.461
Total number of SCF: 14    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -1.2479118031E+00 (Ha/atom)
Total free energy                  : -6.2395590155E+00 (Ha)
Band structure energy              : -2.8711300974E+00 (Ha)
Exchange correlation energy        : -2.4735916939E+00 (Ha)
Self and correction energy         : -1.9373391100E+01 (Ha)
Entropy*kb*T                       : -2.0218005428E-09 (Ha)
Fermi level                        : -2.8359044178E-01 (Ha)
Average force                      :  3.8831802540E-04 (Ha/Bohr)
Maximum force                      :  4.8543678930E-04 (Ha/Bohr)
Time for force calculation         :  4.914 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  498.152 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
